src/audio/copier/copier.c

```diff
@@ -1399,6 +1402,8 @@ static void copier_dma_cb(void *arg, enum notify_id type, void *data)
 	struct comp_dev *dev = arg;
 	struct copier_data *cd = comp_get_drvdata(dev);
 	uint32_t bytes = next->elem.size;
+	int ret, frames;
+	struct comp_buffer __sparse_cache *sink;
```

Comment:
can you please move the buffer declaration before line 1404? to be consustent with our coding style ie structs first?

Reply:
ok, I learn another rule: struct first